src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.props

```diff
@@ -20,6 +20,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ServerGarbageCollection Condition="'$(ServerGarbageCollection)' == ''">true</ServerGarbageCollection>
     <IsPackable Condition="'$(IsPackable)' == ''">false</IsPackable>
     <WarnOnPackingNonPackableProject Condition="'$(WarnOnPackingNonPackableProject)' == '' and '$(IsPackable)' == 'false'">true</WarnOnPackingNonPackableProject>
+    <ShowNativeAOTProperties>false</ShowNativeAOTProperties>
```

Comment:
Why is this set to `false`?

Reply:
At the moment, we don't want to show these properties in web projects like ASP.NET. Is this the right place to set it for those types of projects?

Reply:
Good question for @sayedihashimi. You can find more context on his spec on https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1831643/?triage=true

Reply:
If I remember correctly, ASP.NET already has a couple of properties, so there might be a plan to remove them after this is done.

Reply:
I think you just want to turn this property off for "Web Assembly" / "Blazor WASM" projects. For "normal" ASP.NET Core projects, we definitely want to show Native AOT properties in the property page. We did a bunch of work in .NET 8 to make Native AOT work with ASP.NET Minimal APIs.

Reply:
I'll leave this out of this PR, thanks for the pointer.